client/src/components/map.js

```diff
@@ -26,28 +26,31 @@ const Map = ({ address }) => {
 
   const [location, setLocation] = useState({});
 
-  Geocode.setApiKey(process.env.GOOGLE_API);
-  useEffect(() => {
-    Geocode.fromAddress(address).then(
-      (response) => {
-        setLocation(response.results[0].geometry.location);
-      }).catch((err) => { console.log(err)});
-  }, []);
+  // Geocode.setApiKey(process.env.GOOGLE_API);
```

Comment:
Might want to uncomment these lines back. I think we just did that for debugging?

Reply:
Good catch! Just fixed it.